src/data/polynomial/basic.lean

```diff
@@ -857,6 +861,15 @@ mt (congr_arg (λ p, coeff p 1)) (by simp)
 
 end nonzero_semiring
 
+section division_ring
+
+variables [division_ring R]
+
+lemma rat_smul_eq_C_mul (a : ℚ) (f : R[X]) : a • f = polynomial.C ↑a * f :=
```

Comment:
Do we have the following lemma?
```lean
lemma smul_eq_C_mul (a : S) (f : R[X]) : a • f = polynomial.C (algeba_map _ _ a) * f :=
```
I guess that doesn't work for you here since you're deliberately dealing with the case where the rat-action might be poorly-behaved.

Reply:
I may not have an algebra structure - I'm just dealing with some field where I use this, so if it's e.g. `F_p` the `algebra_map` just doesn't exist.